code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -185,8 +185,5 @@
 /obj/item/weapon/melee/energy/blade/dropped()
 	qdel(src)
 
-/obj/item/weapon/melee/energy/blade/proc/throw()
-	qdel(src)

```

Comment:
@Fox-McCloud it's unused though, since' this is /proc/throw(), a DEFINITION of a throw proc, silly ninja coders and their snowflakeyness